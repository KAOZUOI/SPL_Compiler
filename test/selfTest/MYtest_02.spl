int sum(int x, int y)
{
  int res = x + y + z;
  return res;
}
int test_2_r01()
{
  int a1 = 1;
  int b = a1 + a;
  int res = sum(a, b);
  while (res < 10)
  {
    int xx = 1;
    res = xx;
  }
  res = xx;
  return 0;
}