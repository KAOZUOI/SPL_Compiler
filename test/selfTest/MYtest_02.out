Error type 1 at Line 3: undefined variable "z".
Error type 1 at Line 9: undefined variable "a".
Error type 9 at Line 10: the function "sum" is not applicable for arguments.
Error type 1 at Line 16: undefined variable "xx".